### PR TITLE
Updated to_salesforce_json method of project model

### PR DESCRIPTION
**Pull request by stuartmccoll**
This pull request adds some of the missing model fields to the `to_salesforce_json` method for the `project` model, with dummy data and real data where this is available.

### CODE REVIEW

File: app/models/project.rb [hunk 1]
```diff
@@ -166,6 +166,67 @@ def to_salesforce_json
...
+                    json.county self.county
+                    json.townCity self.townCity
+                    json.projectPostcode self.postcode
+                end
+                json.set!('yourIdeaProject', self.description)
+                json.set!('projectDifference', self.difference)
+                json.set!('projectCommunity', self.matter)
+                json.set!('projectOrgBestPlace', self.best_placed_description)
+                json.set!('projectAvailable', self.heritage_description)
+                json.set!('projectOutcome1', 'Dummy data')
```
Comment (gidsg) [inline app/models/project.rb]:
```suggestion
                json.set!('projectOutcome1', self.involvement_description)
```
This is the projectOuctcome1 question

Comment (gidsg) [reply app/models/project.rb]: Can we update the test to include this field?

**Comment by gidsg** [comment]
Can we update https://github.com/heritagefund/funding-frontend/blob/master/test/models/project_test.rb to check it's pulling through the projectOutcome1 field?

File: app/models/project.rb [hunk 2]
```diff
@@ -166,6 +166,67 @@ def to_salesforce_json
...
+                end
+                json.cashContributions do
+                    json.description 'Dummy data'
+                    json.amount 2000
+                    json.secured 'no'
+                    json.id 'c4237718-ced7-4d03-a95b-1eceaecfdbe0'
+                end
+                json.evidenceOfSupport do
+                    json.description 'Dummy data'
+                end
```
Comment (gidsg) [inline app/models/project.rb]: I need to update the payload but we decided to remove this block as we will associate the data with the file metadata instead. So this should be removed.

File: app/models/project.rb [hunk 3]
```diff
@@ -166,6 +166,67 @@ def to_salesforce_json
...
+                json.set!('projectOutcome6Checked', true)
+                json.set!('projectOutcome7Checked', true)
+                json.set!('projectOutcome8Checked', true)
+                json.set!('projectOutcome9Checked', true)
+                json.projectCosts do
+                    json.costId 'baa49446-cb70-46c7-ade1-0e17ad450c8a'
+                    json.costType 'new-staff'
+                    json.costDescription 'Dummy data'
+                    json.costAmount '1000'
+                end
```
Comment (gidsg) [inline app/models/project.rb]: This should be an array of objects rather than an object.

File: app/models/project.rb (hunk 2 above)
Comment (stuartmccoll) [reply app/models/project.rb]: Done 👍.

File: app/models/project.rb (hunk 3 above)
Comment (stuartmccoll) [reply app/models/project.rb]: Done for all elements that should be arrays containing objects.

**Pull request merged**